operating_systems/opensuse/Dockerfile

```
@@ -0,0 +1,21 @@
+ARG BASEIMAGE=opensuse/leap
+ARG TAG=latest
+
+FROM ${BASEIMAGE}:${TAG}
+ARG UPDATED=false
+
+RUN if [ "$UPDATED" = true ]; then zypper refresh && zypper update -y; fi \
+    && zypper install -y openssh shadow \
```

```suggestion
    && zypper install -y openssh \
```
Isn't `shadow` already pre-installed?

---

Yeah you are right. This is not required to be installed explicitly.